src/components/pages/room.js

```diff
@@ -30,32 +34,35 @@ export default function Room({ PageTemplate, initialState }) {
     window.location.reload();
   };
 
-  const isPrivateRoom = room.access === ROOM_ACCESS_LEVEL.private;
-  const shouldDisplayInvitations = isPrivateRoom && isRoomOwner;
+  const renderLesson = lesson => {
+    const urlParts = ['lessons', encodeURIComponent(lesson._id)];
```

Comment:
This can go to urls.js, see "getDocUrl", it already takes care of a missing slug. If the intention is to not put there a url that doesn't exist yet, you can still used the concatParts method I think.

Reply:
I actually intentionally didn't want to put it there yet, until we discuss with Uli and agree that we want a different page.